Translated Descriptions

DahliaSR opened the pull request:
@GiForce: Hey, pls have a look and let me know what you think about it.

Closes #1

DahliaSR committed:
Translated Short Description

part of #1

DahliaSR committed:
Translated Description

Part of #1

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -1,66 +1,78 @@
+    Die römische Zahlen stammen aus dem alten römischen Zahlensystem.
+    Sie basieren auf bestimmten Buchstaben des Alphabets.
+    Addiert man die Werte der römischen Ziffern, ergibt die Summe (Oder,
+    in einigen Fällen die Differenz) die Zahl.
```
Obviously you like commas 😱I personally think such small one sentence additions flow much better without. They aren't embedded clauses, parentheses even replace commas. Anyway, one thing for sure, lowercase **oder,** and why not the second comma **in einigen Fällen, die Differenz** then ?  Duden online agrees with either way. So do I 😁 One of the rare cases I'm not wrong without commas.

DahliaSR replied:
I **do not** like commas, I just follow the recommendations of Duden. :smile: 
:+1:

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -1,66 +1,78 @@
-    The Roman numeral system is decimal based but not directly positional and does not
-    include a zero. Roman numerals are based on combinations of these seven symbols:
+    Das römische Zahlensystem basiert auf den Zehnerpotenzen als Basis und den
+    fünffachen Halbbasiswerten. Abgesehen von der Subtraktionsregel ist der Wert
```
_fünffachen Halbbasiswerten_ you meant **fünffachen Hilfsbasiswerten**,  didn't you. Or more general and simple: "auf einem Zahlensystem mit den Basiszahlen 5 und 10." Easy,  we have it all from the same Wikipedia entry :)

DahliaSR replied:
It really sounds bad
I like your version and will change it to:
"Das römische Zahlensystem basiert auf den Basiszahlen 5 und 10."
:+1:

DahliaSR committed:
Made Corrections

[Review 97784](https://github.com/CheckIO-to-German/Roman-Numerals/pull/3#pullrequestreview-97784) for PR #3
